frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -107,17 +107,24 @@ class FieldValuesWidgetInner extends Component {
     });
 
     let options = [];
+    let hasIncompleteValueSet;
     try {
       if (usesChainFilterEndpoints(this.props.dashboard)) {
         options = await this.fetchDashboardParamValues(query);
+        // not yet implemented for dashboard parameter values endpoint
+        hasIncompleteValueSet = false;
       } else {
         options = await this.fetchFieldValues(query);
+        hasIncompleteValueSet = this.props.fields.some(
+          field => field.has_more_values,
+        );
       }
     } finally {
       this.updateRemappings(options);
       this.setState({
         loadingState: "LOADED",
         options,
+        hasIncompleteValueSet,
```

Comment:
Instead of relying directly on the `field` property `has_more_values`, I'm storing the value in state on the component, because we will also (eventually) receive a similar property from the dashboard parameter value endpoint, and that won't be stored anywhere (except here, where the request is made).